### PR TITLE
Edits to posters for SciPy 2023

### PR DESCRIPTION
**Pull request by PipKat**
@RobPasMue, @jorgepiloto. @Revathyvenugopal162 
These posters are fabulous--so professional! They will be fantastic contributions to SciPy. My suggested edits are mostly minor. First, I have to confess that I hated the "Any doubts?" phrase and say that it came from your example file. I had to change this phrase to "Any questions?" in your source files and the example file. I also several exclamation points as there were too many for technical content. (The general consensus in all technical style guides are to avoid them. However, there were times I had to allow your enthusiasm to show and so left a couple!)

@jorgepiloto In the left-hand column of your poster, I switched the first word of a section description to a simple verb form rather than the gerund form (a verb ending in "ing") as the subsequent text had lots of gerunds. Plus, to me, starting with a simple verb seemed more powerful.

Lastly, I have a question. I found several occurrences where hyphenation at the end of lines annoyed me. Is there a way to turn hyphenation off or use only left justified text? The only time you see both left and right justification in use these days is in legal documents. The thought there is that it helps prevent someone from trying to change content illegally.

Again, I applaud this excellent work. Concluding my Friday with its review has left a smile on my face!

### CODE REVIEW

**Review by MaxJPRey — APPROVED** [review]
Posters quality is really good.
Kuddos @Revathyvenugopal162 @RobPasMue @jorgepiloto @PipKat

**Comment by PipKat** [comment]
@RobPasMue You are right! It looks better with the hyphenation. There was really only one occurrence that really bothered me. I don't remember which poster it was, but the word was element. It was hyphenated so that "ele-" was on one line and "ment" was on the next. Some programs let you indicate the number of syllables or letters that have to precede a hyphen to avoid such short hyphenations. However, if you can't do that, I agree that leaving hyphenation looks better than turning it off--especially given that it is even stretching bulltet items to the full line length (and that looks bad AND makes it harder to read).

**Review by Revathyvenugopal162 — APPROVED** [review]

File: src/ci_cd.tex [hunk 1]
```diff
@@ -91,16 +91,16 @@ \section{\includegraphics[height=\fontcharht\font`\S]{img/general/slash.png} Tes
 %%%%%%%% RIGHT COLUMN
 
 \section{\includegraphics[height=\fontcharht\font`\S]{img/general/slash.png} Build library}
-Leveraging pre-built components and artifacts to expedite software development,
+Leverages pre-built components and artifacts to expedite software development,
```
Comment (Revathyvenugopal162) [inline src/ci_cd.tex:94]: This and the next section having the same content inside @jorgepiloto , is this expected?

**Comment by RobPasMue** [comment]
> @RobPasMue You are right! It looks better with the hyphenation. There was really only one occurrence that really bothered me. I don't remember which poster it was, but the word was element. It was hyphenated so that "ele-" was on one line and "ment" was on the next. Some programs let you indicate the number of syllables or letters that have to precede a hyphen to avoid such short hyphenations. However, if you can't do that, I agree that leaving hyphenation looks better than turning it off--especially given that it is even stretching bullet items to the full line length (and that looks bad AND makes it harder to read). I'll let you take whatever actions you think best--including the merging of this PR. Great job once again!

Gotcha! Just managed to force LaTeX to at least have 4 letters when performing hyphenation. That should solve our shortly cut words. Thanks once again @PipKat!

File: src/ci_cd.tex (hunk 1 above)
Comment (RobPasMue) [reply src/ci_cd.tex:94]: What do you mean @Revathyvenugopal162?

Comment (RobPasMue) [reply src/ci_cd.tex:94]: Oh Now I understand... Yeah something went wrong here @jorgepiloto - please have a look at it

File: src/ci_cd.tex [hunk 2]
```diff
@@ -91,16 +91,16 @@ \section{\includegraphics[height=\fontcharht\font`\S]{img/general/slash.png} Tes
...
 enhancing efficiency and scalability in the creation of complex applications.
 
 \section{\includegraphics[height=\fontcharht\font`\S]{img/general/slash.png} Release}
-Leveraging pre-built components and artifacts to expedite software development,
+Leverages pre-built components and artifacts to expedite software development,
 enhancing efficiency and scalability in the creation of complex applications.
```
Comment (RobPasMue) [inline src/ci_cd.tex:100]: I think the main issue is that the Release section has just been copy-pasted. Could you adapt it @jorgepiloto?

Comment (RobPasMue) [reply src/ci_cd.tex:100]: Let's do it in a follow up PR

File: src/ci_cd.tex (hunk 1 above)
Comment (PipKat) [reply src/ci_cd.tex:94]: @RobPasMue Looks like it should have this text--which was on the poster here: https://scipy2023.pyansys.com/ci_cd.pdf
Guarantees the creation of components and artifacts for efficient software distribution. Verifies that the artifacts include the proper metadata. 
(Hope I didn't cause the problem!)

Comment (RobPasMue) [reply src/ci_cd.tex:94]: No worries @PipKat! :) we had @jorgepiloto re-write it. It was a mistake on our side in fact. He just opened a PR and we merged it. Thanks anyway!

**Pull request merged**